DashPay/Presentation/Home/DWDashPayReadyProtocol.h

```diff
@@ -19,11 +19,11 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-extern NSNotificationName const DWDashPayAvailabilityStatusUpdatedNotification;
+//extern NSNotificationName const DWDashPayAvailabilityStatusUpdatedNotification; TODO: check if needed
```

Comment:
Never triggered atm. Might not be needed - we might have to wait for a working platform to know for sure.